controller/controller_test.go

```diff
@@ -1361,23 +1367,25 @@ cmd:
 	})
 
 	Describe("ScaleUp", func() {
+		BeforeEach(func() {
+			config.Set("watcher.maxScaleUpAmount", 100)
```

Comment:
Don't we need to unset this value after?

Reply:
Sure, I will change this. I thought config was being reset every test but it was because I was running only one test separately.